website/data/docs-navigation.js

```diff
@@ -120,6 +120,8 @@ export default [
           'service-resolver',
           'service-defaults',
           'proxy-defaults',
+          'ingress-gateway',
```

Comment:
should this list be sorted alphabetically?

Reply:
Yes.

Reply:
Done